src/main/java/com/unb/tracker/TrackerApplication.java

```diff
@@ -1,15 +1,16 @@
 package com.unb.tracker;
 
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
```

Comment:
We should find a way to prevent intelliJ from automatically moving import statements around. Doesn't have to be now, but would be nice in the future.